webextension/scripts/utils.js

```diff
@@ -23,6 +19,50 @@ const newshosts = new Set([
   'www.washingtonpost.com'
 ])
 
+/* * * Browser Detection * * */
+
+function getBrowser() {
+  // the order of these is important!
+  if (navigator.brave) { return 'brave' }
+  else if (navigator.userAgent.indexOf('Edg') !== -1) { return 'edge' }
```

Comment:
Should we detect `indexOf( )` for both 'Edge' (Legacy Version) & 'Edg' (Chromium verison)?

Reply:
Yes, you're correct.